src/AssetBundle.php

```diff
@@ -42,19 +47,79 @@ class AssetBundle
+     * @var array List of CSS files. Each CSS file can be specified in one of the following formats:
      *
-     * Note that only a forward slash "/" should be used as directory separator.
+     * - An absolute URL representing an external asset. For example,
+     *   `https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css` or
+     *   `//cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css`.
+     * - A relative path representing a local asset (e.g. `css/main.css`). The actual file path of a local asset can be
+     *   determined by prefixing {@see $basePath} to the relative path, and the actual URL of the asset can be
+     *   determined by prefixing {@see $baseUrl} to the relative path.
+     * - An array, with the first entry being the URL or relative path as described before, and a list of key/value
+     *   pairs that will be used to overwrite {@see $cssOptions} settings for this entry.
+     *
+     * Note that only a forward slash `/` should be used as directory separator.
+     *
+     * Optionally, use string keys for identifies the CSS file.
+     * If key not set, it will use full CSS file URL as the key.
+     *
+     * Example:
+     *
+     * ```php
+     * public array $css = [
+     *     'https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css',
+     *     'css/main.css',
+     *     ['css/a.css'],
+     *     ['css/b.css', 3],
+     *     ['css/c.css', 3, 'crossorigin' => 'any'],
+     *     'key' => 'css/d.css',
+     * ]
+     * ```
+     *
+     * @psalm-var CssFile[]
      */
     public array $css = [];
 
+    /**
+     * @var array List of CSS blocks. Each CSS block can be specified in one of the following formats:
+     *
+     *  - CSS block. For example, `a { color: red; }`.
+     *  - An array, with the first entry is CSS block, second entry is position on page (integer, optional) that
+     *    will be used to overwrite {@see $cssPosition} setting and a list of key/value pairs
+     *    that will be used to overwrite {@see $cssOptions} settings for this CSS block.
+     *
+     * Optionally, use string keys for identifies the CSS block.
+     *
+     * Example:
+     *
+     * ```php
+     * public array $cssString = [
+     *     'a { color: red; }',
+     *     ['a { color: red; }'],
+     *     ['a { color: red; }', 3],
+     *     ['a { color: red; }', 3, 'crossorigin' => 'any'],
+     *     'key1' => 'a { color: red; }',
+     *     'key2' => ['a { color: red; }'],
+     *     'key3' => ['a { color: red; }', 3],
+     *     'key4' => ['a { color: red; }', 3, 'crossorigin' => 'any'],
+     * ];
+     * ```
+     *
+     * @psalm-var CssString[]
+     */
+    public array $cssStrings = [];
+
     /**
      * @var array The options that will be passed to {@see \Yiisoft\View\WebView::registerCssFile()}
      * when registering the CSS files in this bundle.
      */
     public array $cssOptions = [];
 
+    /**
+     * @var int|null CSS files and blocks position.
```

Comment:
Probably need to specify in docblock what `int ' stands for, what possible values are.

Reply:
Done.